detectron2/data/detection_utils.py

```diff
@@ -97,6 +100,51 @@ def convert_image_to_rgb(image, format):
     return image
 
 
+def apply_exif_orientation(image):
+    """
+    Applies the exif orientation correctly.
+
+    This code exists per the bug:
+      https://github.com/python-pillow/Pillow/issues/3973
+    with the function `ImageOps.exif_transpose`
+
+    Modified from:
+      https://github.com/wkentaro/labelme/blob/v4.5.4/labelme/utils/image.py#L59
+      https://github.com/python-pillow/Pillow/blob/7.1.2/src/PIL/ImageOps.py#L527
+
+    The Pillow source raises errors with various methods, especially `tobytes`
```

Comment:
it seems this line stay with L107-L109 that explains the bug

Reply:
Right, I'll clean this part up then and push soon